include/todo_lib/TodoModel.h

```diff
@@ -0,0 +1,45 @@
+// ===-----------------------------------------------------------------------------------------===
+//
+//  This source file is part of the Todo Command-Line Application open source project
+//
+//  Copyright (c) 2024 Lokesh.T.R, Aadith Jagan Pamarthi and the contributors
+//  Licensed under Apache License v2.0
+//
+//  See https://www.apache.org/licenses/LICENSE-2.0.txt for license information
+//  See https://github.com/lokesh-tr/todo/blob/main/CONTRIBUTORS.txt for the list of contributors
+//
+// ===-----------------------------------------------------------------------------------------===
+
+#ifndef TODOMODEL_H
+#define TODOMODEL_H
+
+#include <string>
+
+class TodoModel {
+private:
+    unsigned int id;
+    static unsigned int nextId;
```

Comment:
Whats this nextId?

Reply:
I've implemented a method where, the program depends on a `static int` for it to change the `id`. This was what I was able to come up with. If there are better alternatives, I'm all aboard to work on implementing it. :thumbs_up